src/components/header.js

```diff
@@ -67,7 +72,11 @@ function Header() {
               title: `Code Of Conduct`,
             },
           ].map(link => (
-            <Link className="navMenuItems" key={link.title} to={link.route}>
+            <Link
+              className={componentStyles.navMenuItems}
+              key={link.title}
```

Comment:
This is neat. I'm trying to deepen my understanding of why braces are used around the values here. I've also seen syntax with dollar symbols placed before them.